ddspipe_core/src/cpp/configuration/CommandlineArgs.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file CommandlineArgs.cpp
+ *
+ */
+
+#include <cpp_utils/utils.hpp>
+
+#include <ddspipe_core/configuration/CommandlineArgs.hpp>
+
+namespace eprosima {
+namespace ddspipe {
+namespace core {
+
+
+CommandlineArgs::CommandlineArgs()
+    : file_path{""},
```

Comment:
NIT: The format we usually follow is:
: file_path{""}
, reload_time{0}
, timeout{0}